backend/python/app/tests/services/test_story_service.py

```diff
@@ -0,0 +1,124 @@
+from ...models.story import Story
+
+
+# https://github.com/apryor6/flask_testing_examples/tree/master/fte
+def test_add_story(app, db):  # noqa
+    # This is not a very useful test, as it basically is testing SqlAlchemy; however,
+    # I leave it here as a reference for how to create objects
+    obj = Story(title="title", description="description", youtube_link="", level=1)
+    db.session.add(obj)
+    db.session.commit()
+    fetched = Story.query.all()
+    assert len(fetched) == 1
+    assert fetched[0].title == "title"
+
+
+def assert_story_equals_model(story_response, story_model):
```

Comment:
would prefer if this is at the top of the file, bottom of the file, or in a util file.
could also be a function written into the Story model! like to_dict

Reply:
Think this is a similar conversation to the test_story_mutation tests and how we want to test models? But also yes, how do we want to handle helpers?